tests/layer_tests/tensorflow_tests/test_tf_BinaryOps.py

```diff
@@ -7,98 +7,86 @@
 import pytest
 from common.tf_layer_test_class import CommonTFLayerTest
 
+rng = np.random.default_rng()
 
-def generate_input(op_type, size):
-    narrow_borders = ["Pow"]
-
-    logical_type = ['LogicalAnd', 'LogicalOr', 'LogicalXor']
-
-    bitwise_type = ['BitwiseAnd', 'BitwiseOr', 'BitwiseXor']
 
+def generate_input(x_shape, x_type, bound=False):
     # usual function domain
-    lower = -256
-    upper = 256
+    lower = -25
```

Comment:
Why do we need to narrow the input range?

Reply:
it is done for some certain operations like Div, Mod for which we can't use zero value for denominator.

Reply:
Yes, I see that zero is excluded in case if '`bound`' is set. But I still don't understand why we narrow [-256,256] to [-25,25] for operations where '`bound`' is not set?

Reply:
For `Pow` it can be giant numbers x^y that will exceed max bound.